docs/create-a-business-component.md

```diff
@@ -0,0 +1,345 @@
+---
+id: create-a-business-component
+title: Create a Business Component
+---
+
+In Front Commerce we have seperated our components in two categories: the **UI**
+components available in the `ui` folders, and the **Business** components
+available in the `modules` and `pages` folders.
+
+> If you feel the need to understand why we went for this organization, feel
+> free to refer to the [components history page](#TODO-lien-history) first.
+
+In this section, we will build a **Business Component**. If you have already
+gone through the [Create an UI component](#TODO-lien-create-ui-component), the
+core concept is the same.
+
+## What is a Business component
+
+A Business component will reside in the `src/theme/modules` directory. Those
+components are not meant to be reused a lot in your application, they are build
+with a **very specific use in mind**. When creating a custom theme, they will
+naturally emerge from your Pages components.
+
+To illustrate this, Imagine that you are building the homepage for your store.
+You add a big hero image on top, some products's listings for news and sales, a
+reinsurance banner, etc.
+
+Quickly, you will want to **extract** some components from your page to avoid a
+_big bloated file_. Some of those components will be extracted as **reusable UI
+components** but some are just very specific to your page and there is no much
+sense to put them in the ui folder.
+
+> They are often a mix between UI components and custom layout. They may be
+> splitted in multiples components if they are big enough.
+
+We put those components in the `src/theme/modules` directory. Generally, they
```

Comment:
This first sentence may be removed since it is already explained a few paragraphs before IMO